src/convertToObject.js

```diff
@@ -3,16 +3,43 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
-}
+  const stylesObject = {};
+
+  sourceString
+    .split(';')
+    .map((element) => element
+      .split(':')
+      .map((element2) => element2.trim())
+    ).forEach((element) => {
+      if (element[0] !== '' && element[0] !== undefined) {
+        stylesObject[element[0]] = element[1];
+      }
```

Comment:
You can use destructuring of an `element` into an array here. `element` is an array consisting of two values: `[key, value]`. It will make this code a bit more readable. You can also filter before the `forEach` to check if there are any empty keys